### PR TITLE
system channel cleanup - remove replicator

### PR DESCRIPTION
**Pull request by semil**
Cleaned dependencies on system channel from the `orderer/common/cluster/replication.go`.

#### Type of change

- Improvement (improvement to code, performance, etc)
- Test update

#### Description

As described in issue #4202 .

#### Related issues

Issue: #4202 .
Related PR: #4207 .

### CODE REVIEW

File: orderer/common/cluster/util.go [hunk 1]
```diff
@@ -48,6 +48,17 @@ type ConnByCertMap map[string]*grpc.ClientConn
 // Lable used for TLS Export Keying Material call
 const KeyingMaterialLabel = "orderer v3 authentication label"
 
+//go:generate mockery --dir . --name LedgerWriter --case underscore --output mocks/
```
Comment (yacovm) [inline orderer/common/cluster/util.go]: I don't think they belong to util. These are definitions.

Let's put them in the file they're used at. which is probably deliver.go.

File: orderer/common/cluster/util.go [hunk 2]
```diff
@@ -208,6 +219,17 @@ type Dialer interface {
 
 var errNotAConfig = errors.New("not a config block")
 
+// ErrForbidden denotes that an ordering node refuses sending blocks due to access control.
+var ErrForbidden = errors.New("forbidden pulling the channel")
```
Comment (yacovm) [inline orderer/common/cluster/util.go]: these belong to deliver.go, not in util

Comment (yacovm) [inline orderer/common/cluster/util.go]: these belong to deliver.go, not in util

File: orderer/common/cluster/util.go (hunk 1 above)
Comment (semil) [reply orderer/common/cluster/util.go]: LedgerWriter is only used by LedgerInterceptor which is in the same file.
Am I missing something?

https://github.com/hyperledger/fabric/blob/c31a8a23ae414a1209c9019bf51120d03a21bded/orderer/common/cluster/util.go#L588

File: orderer/common/cluster/util.go (hunk 2 above)
Comment (semil) [reply orderer/common/cluster/util.go]: moved all the error defs :)

File: orderer/common/cluster/util.go (hunk 1 above)
Comment (yacovm) [reply orderer/common/cluster/util.go]: You're not, I didn't look at the code when reviewing. 

But now when I look at the code (at least, one version of it...)
Is this ledger interceptor in use after removing onboarding?

Comment (semil) [reply orderer/common/cluster/util.go]: You are correct, the LedgerWriter is used by LedgerInterceptor which is only by the test, removed both.

**Review by yacovm — APPROVED** [review]

**Pull request merged**